remappings.txt

```diff
@@ -0,0 +1,4 @@
+forge-std/=lib/forge-std/src/
+openzeppelin-contracts/=lib/openzeppelin-contracts/
+solmate/=lib/solmate/src/
+weird-erc20/=lib/solmate/lib/weird-erc20/src/
```

Comment:
why remap wierd erc20

Reply:
I have no idea where that came from😂

I think it was when I ran `forge remapping > remapping.txt`. 

I'll go remove it now.

Reply:
removed.